App/Sagas/TextileSagas.ts

```diff
@@ -379,3 +379,24 @@ export function * refreshThreads () {
     yield put(ThreadsActions.refreshThreadsError(error))
   }
 }
+
+export function * addExternalInvite (action: ActionType<typeof ThreadsActions.addExternalInvite>) {
+  const { name, pubKey } = action.payload
+  try {
+    const link = yield call(TextileNode.addExternalThreadInvite, name, pubKey)
+    yield put(ThreadsActions.addExternalInviteSuccess(pubKey, link))
+  } catch (error) {
+    yield put(ThreadsActions.addExternalInviteError(error))
+  }
+}
+
+export function * acceptExternalInvite (action: ActionType<typeof ThreadsActions.acceptExternalInvite>) {
+  const { link } = action.payload
+  console.log('ACT', action, action.payload)
```

Comment:
Let's clean up all the console logs if this is working.

Reply:
make sure you refresh